app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script.rb

```diff
@@ -28,21 +28,41 @@ def update_in_provider(manager_id, params)
       EmsRefresh.queue_refresh(manager, nil, true)
     end
 
+    def delete_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      manager.with_provider_connection do |connection|
+        connection.api.job_templates.find(params[:manager_ref]).destroy!
+      end
+
+      # Get the record in our database
+      # TODO: This needs to be targeted refresh so it doesn't take too long
+      EmsRefresh.queue_refresh(manager, nil, true)
```

Comment:
I'll get the change in.  Interestingly enough - since it isn't waiting for a task id the UI treats it like an asynchronous call - but I didn't realize the `true` argument was still hanging around.